azure-pipelines/ui-automation/templates/flank/flank.yml

```diff
@@ -2,7 +2,7 @@ flank:
    ## Max Test Shards
    # test shards - the amount of groups to split the test suite into
    # set to -1 to use one shard per test. default: 1
-   max-test-shards: 3
+   max-test-shards: 2
```

Comment:
can we make this a parameter?

Reply:
I'm inclined to say yes. I originally did not because we can't pass parameters to this flank.yml file (it's not used as a template), which is where flank currently gets it's flank-specific parameters. I can take another look and wait on merging this PR